src/main/java/org/springframework/data/couchbase/core/CouchbaseTemplate.java

```diff
@@ -162,13 +168,16 @@ RuntimeException potentiallyConvertRuntimeException(RuntimeException ex) {
 	@Override
 	public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
 		prepareIndexCreator(applicationContext);
+		templateSupport.setApplicationContext(applicationContext);
+		reactiveCouchbaseTemplate.setApplicationContext(applicationContext);
```

Comment:
I think the reactive template already gets called from spring

Reply:
reactiveCouchbaseTemplate is created and managed by CouchbaseTemplate. 

this.reactiveCouchbaseTemplate = new ReactiveCouchbaseTemplate(clientFactory, converter);